internal/oci/runtime_oci.go

```diff
@@ -45,6 +47,15 @@ const (
 	rootFlag = "--root"
 )
 
+var (
```

Comment:
```suggestion
const (
```

And merge with the block above.

Reply:
Done.